tests/e2e_tests

```diff
@@ -23,9 +24,9 @@ do
 
     if [ -f "run_tests.bash" ]
     then
-        output="$(INFECTION_E2E_TESTS_ENV=1 bash run_tests.bash ${1:-bin/infection})"
+        output="$(INFECTION_E2E_TESTS_ENV=1 bash run_tests.bash ${2:-bin/infection})"
     else
-        output="$(INFECTION_E2E_TESTS_ENV=1 bash ../standard_script.bash ${1:-bin/infection})"
+        output="$(INFECTION_E2E_TESTS_ENV=1 bash ../standard_script.bash ${2:-bin/infection})"
```

Comment:
hmm just noticed it too... we are actually not capturing the stderr at all. In Box I had to start testing both outputs in several cases so I ended up having to output files, but if it doesn't matter we could just combine the stdout and stderr here?

Reply:
done in a separate PR https://github.com/infection/infection/pull/1991